app/src/main/java/com/example/shppprojects/data/database/contactslist/ContactsDbEntity.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.shppprojects.data.database.contactslist
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.example.shppprojects.data.model.Contact
+
+@Entity(
+    tableName = "contacts_list"
+)
+data class ContactsDbEntity(
+    @PrimaryKey val id: Long,
+    val name: String? = null,
+    val career: String? = null,
+    val address: String? = null,
+    val photo: String? = null,
+) {
+    fun toContact() = Contact(
+        name = name,
+        career = career,
+        photo = photo,
+        address = address,
+        id = id
+    )
+
+    companion object {
+        fun toEntity(contact: Contact) = ContactsDbEntity(
```

Comment:
Basically, you shouldn't use static context for creating another object with field of this class